### ISSUE TITLE
False positive "no-invalid-position-at-import-rule" for @use before @import

### ISSUE THREAD

**Issue by prests** [stylelint/stylelint#6715]
### What steps are needed to reproduce the bug?

put an @use statement before an @import

### What Stylelint configuration is needed to reproduce the bug?

```json
{
  "rules": {
    'no-invalid-position-at-import-rule': true
  }
}
```

### How did you run Stylelint?

cli with stylelint "**/*.{scss,vue}"

### Which version of Stylelint are you using?

15.3.0

### What did you expect to happen?

Allow @use to come before @import in scss file

### What actually happened?

Unexpected invalid position @import rule  no-invalid-position-at-import-rule

### Does the bug relate to non-standard syntax?

Yes related to SCSS

**Comment by jeddy3** [comment]
@prests Thanks for the report and for using the template. 

The built-in rules and official configs are geared towards CSS. You should [extend a community config](https://stylelint.io/user-guide/get-started#linting-css-like-languages-and-css-within-containers), when linting a CSS-like language like SCSS.

The [stylelint-config-standard-scss](https://www.npmjs.com/package/stylelint-config-standard-scss) config by the [Stylelint SCSS community](https://github.com/stylelint-scss) configures the `no-invalid-position-at-import-rule` rule to be [compatible with SCSS](https://github.com/stylelint-scss/stylelint-config-recommended-scss/blob/master/index.js#L12-L17).

If you're linting Vue and SCSS, you can use:

```json
{
  "extends": [
    "stylelint-config-standard-scss",
    "stylelint-config-standard-vue/scss"
  ]
}
```
